src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const cost = days * 40;
+
+  if (days < 3) {
+    return cost;
+  } else if (days < 7) {
+    return cost - 20;
```

Comment:
```suggestion
    return (cost - 20);
```
use brackets with expressions